lib/class-wp-duotone-gutenberg.php

```diff
@@ -860,15 +901,44 @@ public static function migrate_experimental_duotone_support_flag( $settings, $me
 		return $settings;
 	}
 
+	/**
+	 * Returns the prefixed id for the duotone filter for use as a CSS id.
+	 *
+	 * @param  array $preset Duotone preset value as seen in theme.json.
+	 * @return string        Duotone filter CSS id.
+	 */
+	public static function get_filter_id_from_preset( $preset ) {
+		$filter_id = '';
+		if ( isset( $preset['slug'] ) ) {
+			$filter_id = self::get_filter_id( $preset['slug'] );
+		}
+		return $filter_id;
+	}
+
 	/**
 	 * Gets the SVG for the duotone filter definition from a preset.
 	 *
 	 * @param array $preset The duotone preset.
 	 * @return string The SVG for the filter definition.
 	 */
 	public static function get_filter_svg_from_preset( $preset ) {
-		// TODO: This function will be refactored out in a follow-up PR where it will be deprecated.
-		$filter_id = gutenberg_get_duotone_filter_id( $preset );
+		$filter_id = self::get_filter_id_from_preset( $preset );
```

Comment:
Should we have a deprecated call in here?

Reply:
It looks like we do that here https://github.com/WordPress/gutenberg/pull/49705/files#diff-c6cf688aa7b50316afa7fc57327b6ac81c1ac831226447c53ffc88c32fc23750R1009